### PR TITLE
[RW-7076][risk=no] adding cdr date in subselect

### PR DESCRIPTION
**Pull request by freemabd**
Remove the csv/table dependency for cdr date into a subselect.

### CODE REVIEW

File: api/libproject/devstart.rb [hunk 1]
```diff
@@ -849,34 +849,6 @@ def make_prep_ppi_csv_files(cmd_name, *args)
   :fn => ->(*args) { make_prep_ppi_csv_files("make-prep-ppi-csv-files", *args) }
 })
 
-def make_prep_tables_from_csv(cmd_name, *args)
```
Comment (freemabd) [inline api/libproject/devstart.rb:852]: This project.rb command is no longer needed.

File: api/db-cdr/generate-cdr/validate-prerequisites-exist.sh [hunk 2]
```diff
@@ -7,14 +7,12 @@ export BQ_PROJECT=$1        # CDR project
 export BQ_DATASET=$2        # CDR dataset
 export CDR_VERSION=$3       # CDR version
 
-PREP_TABLE_RUN="!_prep_tables_!"
```
Comment (freemabd) [inline api/db-cdr/generate-cdr/validate-prerequisites-exist.sh:10]: This was something Jon used for make-prep-tables-from-csv which is no longer needed.

File: api/db-cdr/generate-cdr/validate-prerequisites-exist.sh [hunk 3]
```diff
@@ -61,28 +58,6 @@ DEPENDENT_TABLES=("activity_summary"
             "vocabulary")
 INCOMPATIBLE_DATASETS=("R2019Q4R3" "R2019Q4R4")
 
-function removeHeaderIfExist() {
```
Comment (freemabd) [inline api/db-cdr/generate-cdr/validate-prerequisites-exist.sh:64]: Non of the csv files have headers, so this is no longer needed.

File: api/db-cdr/generate-cdr/validate-prerequisites-exist.sh [hunk 4]
```diff
@@ -95,19 +70,6 @@ function loadCSVFile() {
   echo "Finished loading $file"
 }
 
-function validateCutOffDate() {
```
Comment (freemabd) [inline api/db-cdr/generate-cdr/validate-prerequisites-exist.sh:98]: No longer needed since we get the cutoff date from the subselect added.

**Review by NehaBroad — APPROVED** [review]

**Pull request merged**